src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -846,6 +906,16 @@ public void initBinder(WebDataBinder webdataBinder) {
             .then(studyService.createEquipment(studyUuid, createVoltageLevelAttributes, ModificationType.VOLTAGE_LEVEL_CREATION, nodeUuid)));
     }
 
+    @PutMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/network-modification/voltage-levels")
```

Comment:
Add Voltage-levesUUid as pathVariable for PUT request
same for all new PUT requests